.gitignore

```diff
@@ -0,0 +1,5 @@
+/test.md
+\test.md
+/node_modules
+\node_modules
+test.md
```

Comment:
- **[OPTIONAL]** The correct name of the file is `./test.md` please 🙏🏽 correct it.